node/router/messages/src/peer_response.rs

```diff
@@ -39,20 +39,56 @@ impl ToBytes for PeerResponse {
             return Err(io::Error::new(io::ErrorKind::InvalidInput, format!("Too many peers: {}", self.peers.len())));
         }
 
+        // A version indicator; we don't expect empty peer responses, so a zero value can serve
+        // as an indicator that this message is to be processed differently. The version value
+        // can be changed to a 2 in the future, once everyone expects it there.
+        0u8.write_le(&mut writer)?;
+
         (self.peers.len() as u8).write_le(&mut writer)?;
-        for peer in self.peers.iter() {
-            peer.write_le(&mut writer)?;
+        for (addr, height) in self.peers.iter() {
+            addr.write_le(&mut writer)?;
+            if let Some(h) = height {
+                1u8.write_le(&mut writer)?;
+                h.write_le(&mut writer)?;
+            } else {
+                0u8.write_le(&mut writer)?;
+            }
         }
         Ok(())
     }
 }
 
 impl FromBytes for PeerResponse {
     fn read_le<R: io::Read>(mut reader: R) -> io::Result<Self> {
-        let count = u8::read_le(&mut reader)?;
+        // Read the peer count if their heights aren't present; otherwise, interpret this value
+        // as the message version. It is a workaround for a currently missing version value.
```

Comment:
I think it's worth also documenting the worst case scenario. If I understand correctly: for nodes which don't upgrade, and which have just a single peer, that peer may not be able to deserialize the empty `PeerResponse`. It would be nice to have a testcase for the success and failure cases, perhaps at either of these places: 
- https://github.com/ProvableHQ/snarkOS/blob/mainnet/node/router/messages/src/helpers/codec.rs#L123
- https://github.com/ProvableHQ/snarkOS/tree/mainnet/node/router/messages/src/lib.rs#L336